doc/md/language-manual.md

```diff
@@ -1367,6 +1367,15 @@ The declaration `let <pat> = <exp>` evaluates `<exp>` to a result `r`. If `r` is
 
 All bindings declared by a `let` (if any) are *immutable*.
 
+#### Handling pattern match failures
+
+In the presence of of refutable patterns a `let` declaration may fail to bind the expression. The default consequence of such a failure is trapping. The compiler will additionally emit a warning if a pattern match failure is a possibility.
+
+There are cases, however, when the user wants to explicitly handle such pattern match failures. For such cases
+the let declaration `let <pat> = <exp> else <fail-expr>` is provided, and has identical static and dynamic semantics with
```

Comment:
```suggestion
the let declaration `let <pat> = <exp> else <fail-expr>` is provided, and has identical static and dynamic semantics with ....
```
with what?

Reply:
I could swear I did complete this sentence. Let me fix.

Reply:
Maybe you reviewed before I finished it.